variables.tf

```diff
@@ -102,3 +102,8 @@ variable "tfc_url" {
   description = "The Terraform Cloud endpoint.  Must be changed if using Terraform Enterprise."
 }
 
+variable "annotations" {
+  description = "Add annotations for service accounts"
```

Comment:
It would be good to mention that irsa use case in the description

Reply:
Changes incorporated as suggested to the README, variable name and description . Please review @redeux .